components/docs-chef-io/content/automate/install.md

```diff
@@ -168,6 +168,7 @@ Add the following to your config.toml:
 
 Because externally-deployed Elasticsearch nodes will not have access to Chef Automate's built-in backup storage services, you must configure Elasticsearch backup settings separately from Chef Automate's primary backup settings. You can configure backups to use either the local filesystem or S3.
 
+__Please note that Chef Automate does not support Elastic Search services. Automate is not tested against Elasticsearch services provide by cloud provides like AWS Elasticsearch Service.__
```

Comment:
```suggestion
__Please note that Chef Automate does not support external Elastic Search services. Automate is not tested against Elasticsearch services provide by cloud provides such as AWS Elasticsearch Service.__
```

Reply:
If we don't support external Elasticsearch services, what do we support? How should users configure analytics and search in their systems?

Reply:
By External elasticsearch services I mean the services provided by cloud providers like AWS. External elasticsearch deployment done and managed by customer over EC2 instances/VMs/bare-bone servers will be still supported.

Reply:
What can be the comment to put it in a clearer way?